### ISSUE TITLE
It's a little broken

### ISSUE THREAD

**Issue by glibg10b** [TRPB/FewerFox#2]
Collapsed:

![image](https://user-images.githubusercontent.com/56197853/168481350-ff590fe0-5ddc-421e-81f2-f3bdb1613b11.png)

Expanded (looks fine):

![image](https://user-images.githubusercontent.com/56197853/168481361-e8999697-abeb-46d2-8689-4e1b67e01f6e.png)

Linux, `firefox-91.9.0:esr`, 1440p

It looks the same in a private window where extensions are disabled

**Comment by TRPB** [comment]
One of those things is easy to fix: It's cutting off the G of github. Unfortunately, it moves the URL to the left to hide `https://` and how many pixels/ems that is will be font dependent. I'll add a variable for tweaking the amount it moves it, but it's going to have to be a manual change in the code as everyone will be using slightly different fonts.

The address bar itself being off the screen is slightly trickier though could again be a font size difference. I'll rework it to use `vw` for that calculation and it should be the same for everyone.

**Comment by TRPB** [comment]
I think I've found the cause. My calculation assumed there are 3 buttons to the left of the URL bar. I've added an update with some variables at the top of the script

You'll need to set `--buttons-left-of-urlbar: 2;`  to the number of buttons you have visible and probably need some trial and error with some minor adjustments for `--https-text-offset-amount` to stop part of the first character getting cut off, or set it to 0 if you don't mind `https://` still using up space in the url bar.
